resources/migrations/000_migrations.yaml

```diff
@@ -14378,6 +14378,22 @@ databaseChangeLog:
                   constraints:
                     nullable: false
 
+  - changeSet:
```

Comment:
The BE change isn't meant to be merged, this will be here until it's properly implemented by BE team.

Reply:
So @dpsutton implemented the BE portion in this BE, now this code is relevant and would be merged to `master`.